cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -126,20 +126,21 @@ describe("User Creation", () => {
   });
 
   it("Update the existing user profile Form Mandatory File Error", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
-    userCreationPage.clickElementById("edit-cancel-profile-button");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    userCreationPage.clickEditCancelProfileButton();
+
     userCreationPage.clearIntoElementById("firstName");
```

Comment:
remove the usage of clearintoelementbyid and use individual proper function for each action with POM approach